templates/web-deployment.yaml

```diff
@@ -754,9 +754,9 @@ spec:
             - name: CONCOURSE_X_FRAME_OPTIONS
               value: {{ .Values.concourse.web.xFrameOptions | quote }}
             {{- end }}
-            {{- if .Values.concourse.web.ContentSecuritPolicy }}
+            {{- if .Values.concourse.web.contentSecuritPolicy }}
             - name: CONCOURSE_CONTENT_SECURITY_POLICY
-              value: {{ .Values.concourse.web.ContentSecuritPolicy | quote }}
+              value: {{ .Values.concourse.web.contentSecuritPolicy | quote }}
```

Comment:
Looks like this (and above) still has the `Securit` typo (missing the "y")

Reply:
CAN YOU TELL I NEED MORE SLEEP 🙃

Reply:
This time I used `ag` to make sure I caught all my typos!